examples/opf/experiments/multistep/datasets/first_order_0.csv

```diff
@@ -2,14 +2,11 @@ reset,field1,field2
 int,string,float
 R,,
 1,0,0
-0,1,1
-0,2,2
-0,0,0

```

Comment:
Why all the changes to this file? I'm not certain this was meant to be included in this PR.